Change process based on feedback

terrajobst opened the pull request:
We intended to close issues after 30 days (#42) but the overwhelming feedback seems to be that people prefer issues remain open in perpetuity.

Commit:
Change process based on feedback

Commit:
List benefits of not closing issues

akoeplinger commented on README.md:
```diff
@@ -4,7 +4,14 @@ This repo is for team announcements only. To file a bug or start a discussion, p
+* You can easily search the announcements and filter without having to consider closed issues
 
 ## Receiving Notifications
 
```
While you're at it please also remove the `Announcement` label search link in "Using the Issues list" below since it doesn't seem to be used in this repo.

terrajobst replied:
Fixed

af4jm replied:
the `Breaking Change` link seems similarly unused.

terrajobst replied:
They are used -- I just happen to have closed them as part of the old process. Once the PR is merged, I'm going to reopen the closed ones too.

Commit:
Update links to labels

Commit:
Remove .gitignore

Since we're not adding source files having an ignore file feels like a
senseless deed. Also, makes the root more busy than it needs to be.

Hygiene & aesthetics: they matter.

kevindqc commented on README.md:
```diff
@@ -1,5 +1,9 @@
 # Announcements
 
+asdsadsad
```
Agreed, but what about people not using qwerty?

terrajobst replied:
OMG. I feel dumb right now. Fixed with 69b11eb9412e18ea438277a66e87a61f95d49b0e.
